score-lang-cli/src/main/java/org/openscore/lang/cli/SlangCLI.java

```diff
@@ -72,9 +73,9 @@ public String run(
         if (!triggerAsync) {
             StopWatch stopWatch = new StopWatch();
             stopWatch.start();
-            id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties);
+            id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties, quiet);
             stopWatch.stop();
-            return triggerSyncMsg(id, stopWatch.toString());
+            return (!quiet) ? triggerAsyncMsg(id, compilationArtifact.getExecutionPlan().getName()) : "";
         }
         id = scoreServices.trigger(compilationArtifact, inputs, systemProperties);
         return triggerAsyncMsg(id, compilationArtifact.getExecutionPlan().getName());
```

Comment:
I only added the option for the synchronous execution. You said in #64 that the asynchronous execution doesn't print tasknames, so quiet mode wouldn't affect it, would it?